rasterio/_base.pyx

```diff
@@ -534,39 +528,44 @@ cdef class DatasetReader(object):
         return self._transform
 
     property transform:
-        """Coefficients of the affine transformation that maps col,row
-        pixel coordinates to x,y coordinates in the specified crs. The
-        coefficients of the augmented matrix are shown below.
+        """An instance of ``affine.Affine``, which is a ``namedtuple`` with
+        coefficients in the order ``(a, b, c, d, e, f)``.
+
+        Coefficients of the affine transformation that maps ``col,row``
+        pixel coordinates to ``x,y`` coordinates in the specified crs. The
+        coefficients of the augmented matrix are:
         
           | x |   | a  b  c | | r |
           | y | = | d  e  f | | c |
           | 1 |   | 0  0  1 | | 1 |
-        
-        In Rasterio versions before 1.0 the value of this property
-        is a list of coefficients ``[c, a, b, f, d, e]``. This form
-        is *deprecated* beginning in 0.9 and in version 1.0 this 
-        property will be replaced by an instance of ``affine.Affine``,
-        which is a namedtuple with coefficients in the order
-        ``(a, b, c, d, e, f)``.
-
-        Please see https://github.com/mapbox/rasterio/issues/86
-        for more details.
         """
         def __get__(self):
-            warnings.warn(
-                    "The value of this property will change in version 1.0. "
-                    "Please see https://github.com/mapbox/rasterio/issues/86 "
-                    "for details.",
-                    FutureWarning,
-                    stacklevel=2)
-            return self.get_transform()
+            return Affine.from_gdal(*self.get_transform())
```

Comment:
Should `get_transform` be modified to return an `Affine` instead?   This seems inconsistent.